Error on make

Trying to run the script, but on `make` I get this:

```
mv temp-logins.json old-logins.json
mv: rename temp-logins.json to old-logins.json: No such file or directory
make: *** [1] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,12 @@
+all: get format
+get: 1 2
+format: 3
+
 1:
 	if [ -e temp-logins.json ]; then mv temp-logins.json old-logins.json; fi;
 	coffee get-users.coffee
-	if [ -e raw/github-users-stats.json ]; then coffee check-logins.coffee; fi;
+	# for debug - requires get-users.coffee/get-details.coffee already ran:
+	#coffee check-logins.coffee
 
 2:
 	coffee get-details.coffee
@@ -24,3 +29,9 @@ force-sync-raw:
 sync-formatted:
 	cd formatted && git commit -am 'Sync.' --amend && git push --force
 
+clean:
+	rm temp-logins.json
+	rm old-logins.json
+	rm raw/github-languages-stats.json
+	rm raw/github-users-stats.json
+	rm formatted/active.md

--- a/get-users.coffee
+++ b/get-users.coffee
@@ -13,8 +13,10 @@ DISQUALIFIED = [
 saveTopLogins = ->
   MIN_FOLLOWERS = 255
   MAX_PAGES = 10
-  urls = utils.range(1, MAX_PAGES + 1).map (page) ->
-    "https://api.github.com/legacy/user/search/followers:%3E#{MIN_FOLLOWERS}?sort=followers&order=desc&start_page=#{page}"
+  urls = utils.range(1, MAX_PAGES + 1).map (page) -> [
+      "https://api.github.com/legacy/user/search/followers:%3E#{MIN_FOLLOWERS}",
+      "?sort=followers&order=desc&start_page=#{page}"
+    ].join('')
 
   parse = (text) ->
     JSON.parse(text).users.map (_) -> _.username

--- a/utils.js
+++ b/utils.js
@@ -17,7 +17,7 @@ var batchGet = exports.batchGet = function(urls, progressback, callback) {
             if (response.status === 404) {
               done();
             } else {
-              throw response.error;
+              throw [response.error,response.text].join("\n");
             }
           }
           var result;

